### PR TITLE
solc-select: init at 0.2.1

### PR DESCRIPTION
**Pull request by arcz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -8150,6 +8150,8 @@ in {
 
   solax = callPackage ../development/python-modules/solax { };
 
+  solc-select = callPackage ../development/python-modules/solc-select { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:8153]: Is this a library or a cli tool? If the later we do not need an entry here.

File: pkgs/development/python-modules/solc-select/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  pname = "solc-select";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-6VawTcffIgnR+zuC4rti+Ocwu1VMTX+VihT/L7LzchI=";
+  };
+
+  # no tests
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/solc-select/default.nix:16]: Please add a pythonImportsCheck.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131943` at 83c3f03d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.solc-select</li>
    <li>solc-select (python39Packages.solc-select)</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/solc-select/default.nix:19:0:
```
   |
19 |     description = "Manage and switch between Solidity compiler versions";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/solc-select/default.nix (hunk 2 above)
Comment (arcz) [reply pkgs/development/python-modules/solc-select/default.nix:16]: added

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (arcz) [reply pkgs/top-level/python-packages.nix]: I removed it for now, can be readded if something depends on it

**Review by SuperSandro2000 — COMMENTED** [review]
You are missing an entry in python-packages.

**Pull request merged**